docusaurus/docs/reference/tunnelers/linux/linux-tunnel-troubleshooting.md

```diff
@@ -23,29 +23,105 @@ journalctl -xeu ziti-edge-tunnel.service
 
 ## Intercepting or hosting not working
 
-You may inspect the loaded identity and router info for a running `ziti-edge-tunnel` by dumping it to stdout or the systemd journal with an IPC command, or you may signal to dump the identities' info to a file.
+You may inspect the loaded identity and router info for a running `ziti-edge-tunnel` by dumping it to stdout or the systemd journal with an IPC command. Alternatively, you may signal to dump the identities' info into a file.
+
+```bash
+# dump identities info to std our journal if systemd unit with IPC command
+./ziti-edge-tunnel dump
+```
+
+```bash
+# dump identities info to a file and inspect
+sudo pkill -USR1 -f ziti-edge-tunnel
+less /tmp/ziti-dump.964315.dump
+```
 
-  ```bash
-  # dump identities info to std our journal if systemd unit with IPC command
-  ./ziti-edge-tunnel dump
-  ```
+What is the full command-line for the running process?
```

Comment:
should be factual, not conversational

Reply:
> is this full/complete for any/all linux variants?

`pgrep -f ziti-edge-tunnel|xargs ps -fww` is broadly-applicable for modern Linuxes, not a particular distro family or ultra-recent vintage.